internal/services/oracledatabase/cloud_vm_cluster_data_source.go

```diff
@@ -0,0 +1,526 @@
+	TimeZone                     string                       `tfschema:"time_zone"`
+	VipIds                       []string                     `tfschema:"vip_ods"`
+	VnetId                       string                       `tfschema:"vnet_id"`
+	ZoneId                       string                       `tfschema:"zone_id"`
+}
+
+type SystemDataModel struct {
+	CreatedBy          string `tfschema:"created_by"`
+	CreatedByType      string `tfschema:"created_by_type"`
+	CreatedAt          string `tfschema:"created_at"`
+	LastModifiedBy     string `tfschema:"last_modified_by"`
+	LastModifiedbyType string `tfschema:"last_modified_by_type"`
+	LastModifiedAt     string `tfschema:"last_modified_at"`
+}
+
+type DataCollectionOptionsModel struct {
+	IsDiagnosticsEventsEnabled bool `tfschema:"is_diagnostics_events_enabled"`
+	IsHealthMonitoringEnabled  bool `tfschema:"is_health_monitoring_enabled"`
+	IsIncidentLogsEnabled      bool `tfschema:"is_incident_logs_enabled"`
+}
+
+type ExadataIormConfigModel struct {
+	DbPlans          []DbIormConfigModel `tfschema:"db_plans"`
+	LifecycleDetails string              `tfschema:"lifecycle_details"`
+	LifecycleState   string              `tfschema:"lifecycle_state"`
+	Objective        string              `tfschema:"objective"`
+}
+
+type DbIormConfigModel struct {
+	DbName          string `tfschema:"db_name"`
+	FlashCacheLimit string `tfschema:"flash_cache_limit"`
+	Share           int64  `tfschema:"share"`
+}
+
+func (d CloudVmClusterDataSource) Arguments() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"resource_group_name": commonschema.ResourceGroupNameForDataSource(),
+		"name": {
+			Type:     pluginsdk.TypeString,
+			Required: true,
+		},
+	}
+}
+
+func (d CloudVmClusterDataSource) Attributes() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"location": commonschema.LocationComputed(),
+		"type": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"tags": commonschema.TagsDataSource(),
+
+		// SystemData
+		"system_data": {
+			Type:     pluginsdk.TypeList,
+			Computed: true,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"created_by": {
+						Type:     pluginsdk.TypeString,
+						Computed: true,
+					},
+					"created_by_type": {
+						Type:     pluginsdk.TypeString,
+						Computed: true,
+					},
+					"created_at": {
+						Type:     pluginsdk.TypeString,
+						Computed: true,
+					},
+					"last_modified_by": {
+						Type:     pluginsdk.TypeString,
+						Computed: true,
+					},
+					"last_modified_by_type": {
+						Type:     pluginsdk.TypeString,
+						Computed: true,
+					},
+					"last_modified_at": {
+						Type:     pluginsdk.TypeString,
+						Computed: true,
+					},
+				},
+			},
+		},
+
+		// CloudVMClusterProperties
+		"backup_subnet_cidr": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+		"cloud_exadata_infrastructure_id": {
```

Comment:
there should be a new line here
```suggestion
		},

		"cloud_exadata_infrastructure_id": {
```

Reply:
Done, thanks!